test/kubemark/pre-existing/README.md

```diff
@@ -0,0 +1,45 @@
+# Kubemark Pre-existing Provider Guide
+
+**Pre-existing Kubernetes Cluster**
```

Comment:
How about something like `Pre-existing kubemark master` as this is a bit confusing? Because I was telling about this to someone the other day and they too mistook it for a pre-existing base cluster (which is anyway true for all scenarios of kubemark).

Also just for clarity, are you referring to using the same master as that of your minions as your kubemark master too?